index.bs

```diff
@@ -623,6 +627,16 @@ A {{WebTransport}} object has the following internal slots.
    throughput or low latency was requested by the application and satisfied
    by the user agent, or `"default"`.
   </tr>
+  <tr>
+   <td><dfn>`[[RateControlFeedbackMinInterval]]`</dfn>
+   <td class="non-normative">A {{long}} indicating the maximum frequency
+   that the {{WebTransport/ratecontrolfeedback}} will be fired.
```

Comment:
Isn't this the minimum time between events?  Or even the duration over which the event is suppressed after receiving another.

Sure interval is inversely proportional to rate, but only loosely.